src/openrct2/CmdlineSprite.cpp

```diff
@@ -640,9 +641,10 @@ int32_t cmdline_for_sprite(const char** argv, int32_t argc)
 
         bool silent = (argc >= 4 && strcmp(argv[3], "silent") == 0);
 
-        spriteFile.Header.num_entries = 0;
-        spriteFile.Header.total_size = 0;
-        sprite_file_save(spriteFilePath);
+        SpriteFile baseFile;
+        baseFile.Header.num_entries = 0;
+        baseFile.Header.total_size = 0;
+        baseFile.Save(spriteFilePath);
```

Comment:
(On commit https://github.com/OpenRCT2/OpenRCT2/pull/13590/commits/3b39ab2299e88ae7cee242006590ab3ec0415b8f)

Shouldn't this be using the `static` variable `spriteFile`?

Reply:
no it is localised to just this branch of the if no need to be the global